src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -341,16 +341,24 @@ private AIState chopTree()
             }
         }
 
-        if (!job.tree.hasLogs() && (!job.tree.isSlimeTree() || !job.tree.hasLeaves()))
-        {
-            if (hasNotDelayed(WAIT_BEFORE_SAPLING))
-            {
+        if (!job.tree.hasLogs() && (!job.tree.isSlimeTree() || !job.tree.hasLeaves())) {
+            if (hasNotDelayed(WAIT_BEFORE_SAPLING)) {
                 return getState();
             }
-            plantSapling();
-            this.getOwnBuilding().getColony().getStatsManager().incrementStatistic("trees");
-            workFrom = null;
-            return LUMBERJACK_GATHERING;
+
+            final AbstractBuilding building = getOwnBuilding();
+            if (((BuildingLumberjack) building).shouldReplant())
+            {
+                plantSapling();
+            }
+            else
+            {
+                job.tree = null;
+                checkedInHut = false;
+            }
+                building.getColony().getStatsManager().incrementStatistic("trees");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)